### ISSUE TITLE
Add README

### ISSUE THREAD

**Issue by melvincarvalho** [solid/profile-viewer-tutorial#1]
I think a README would be helpful

What could be made clearer is the mechanism to create [solid-auth-client.bundle.js](https://github.com/solid/profile-viewer-tutorial/blob/master/scripts/solid-auth-client.bundle.js)

**Comment by RubenVerborgh** [comment]
> I think a README would be helpful

Definitely. As you can see, this code was put together right before showtime; I'll try to document what we wrote.

> What could be made clearer is the mechanism to create solid-auth-client.bundle.js

No one should have to create it, really, but just download it from somewhere.
For now, the instructions are:

```bash
$ npm install solid-auth-client
$ node_modules/solid-auth-client/dist-lib/solid-auth-client.bundle.js .
```

**Comment by RubenVerborgh** [comment]
> just download it from somewhere.

Now auto-publishing at https://solid.github.io/solid-auth-client/dist/solid-auth-client.bundle.js

**Comment by yanyanhu** [comment]
Hi, any update on the README file? I'm trying the Solid App setup [tutorial](https://solid.inrupt.com/docs/app-on-your-lunch-break
). It would be really helpful if there is a bit more detail about the usage of this code, especially for people who have no experience on frontend development, like me :) Thanks!
